run.sh

```diff
@@ -1,30 +1,45 @@
 #!/bin/bash
-set -euxo pipefail
+set -euo pipefail
+
+function outputResult() {
+  local result=$1
+  local exitCode=$2
+
+  echo $result
+
+  if [ $OUTPUT_RESULT = true ]; then
+    echo "::set-output name=result::$result"
```

Comment:
Do you not need to include the `exitCode` here? I do not understand what is happening here.

Maybe what you mean is that `result` is either `passed` or some sort of failure message? In that case there should be no `exitCode` arg, and…

Reply:
added, the script should always exit 0 if the exit code is being set as an output